trane/datasets/data/store/README.md

```diff
@@ -0,0 +1,17 @@
+# Dell Store 2
```

Comment:
Add these files to here:
https://github.com/trane-dev/Trane/blob/3f2b256e3acd44c43ab03a7f3cff3e6d4d54893f/pyproject.toml#L81